### ISSUE TITLE
19 Nov: Intro to LaTeX

### ISSUE THREAD

**Issue by murraycadzow** [smilefreak/studyGroup#42]
An introductory lesson to creating documents using LaTeX

**Comment by murraycadzow** [comment]
You will need to download and install LaTeX to be able to use it

Find your operating system version here and follow the instructions
https://tug.org/texlive/

**Comment by smilefreak** [comment]
I recommend mactex for mac users.
https://tug.org/mactex/

**Comment by murraycadzow** [comment]
@rikutakei would you be happy taking this lesson?

**Comment by methylnick** [comment]
sorry to hijack, is it worth learning latex in the context of Authorea? Latex is powerful, but 80-90% of the users will use only a handful of the functionality, Authorea makes this a little easier and well, also demonstrates the collaborative aspect. Just planting the seed

**Comment by murraycadzow** [comment]
I think the main goal is to introduce LaTex so the actual context is quite flexible
The main reason we in particular would use LaTeX is to (hopefully) make the write up of a thesis easier but it is useful to be able to use it in Rmarkdown documents.

Authorea would certainly be useful for the context of writing papers

**Comment by smilefreak** [comment]
Authorea is certainly worth having a session on, but I think it could confuse teaching the absolute latex basics @methylnick, For many people latex is the thing they were told to write their thesis in, but never did. It's not hard to motivate it that way as after writing one long document in word, you want to end it all. Minimalist teaching environments are great for first-timers, extra complexity and options mostly make it harder to teach and make your points clear. For example, Rstudio makes teaching programming in R harder, but we can't avoid it because it's just so damn good.

In short, not for this one, but next year let's do a authorea one focusing on how to write a paper and using the extra features it adds to latex.

**Comment by methylnick** [comment]
sounds like a plan @smilefreak

**Comment by rikutakei** [comment]
If the lesson is on the 19th I won't be able to do it - I won't be back in time... Sorry

**Comment by murraycadzow** [comment]
Is there anyone else that would like to teach us about LaTex? Otherwise I would suggest we switch this session with the R sub setting session

**Comment by smilefreak** [comment]
Maybe switch to dataframe subsetting if we can't find anyone. It would be nice is someone else would volunteer @murraycadzow  you have been doing lots of these things lately.

**Comment by murraycadzow** [comment]
There would also be scope for people to share parts of a lesson

Eg for LaTex, someone could teach a basic document setup, someone else could cover putting in references and cross-references

or in the subsetting context: usage of subset(), usage of [ ] and usage of filter() from dplyr

**Comment by smilefreak** [comment]
Yea, I think some R related would be best - only a few people know latex AFAIK - so it would be harder to find people. What about Anthony? Van? I have asked MIk to ask the group at the lab meeting today, will see how that goes.

**Comment by murraycadzow** [comment]
There is this very recent resource I made that people can use/modify that contains at least some basics https://github.com/murraycadzow/R_tutorials/blob/master/Intro_R.Rmd

**Comment by TomKellyGenetics** [comment]
I think sticking with LaTeX rather than Authorea is the way to go for an intro. Shouldn't be too hard to switch the other way and the power of LaTeX for postgrads is definitely how it formats (even very large) documents for you. Although Authorea, BibTeX, and Journal paper templates are also cool stuff.

**Comment by TomKellyGenetics** [comment]
Mik mentioned in yesterday's meeting that he may be able to run the LaTeX session if Riku isn't back in time. Probably easier to swap with the R session though, we have done basic programming / data handling in a while anyway. Would be easier to find someone able to do R too. I'd offer to but you guys have probably heard enough from me lately with 3 presentations this month already.

**Comment by TomKellyGenetics** [comment]
If we're doing LaTeX, either TeXstudio or MikTeX are both ok for Windows and pretty easy to install. I use Kile for Linux but if you're a Linux user you probably know you're favourite tex distro and IDE already.

**Comment by murraycadzow** [comment]
**This lesson has been rescheduled to be on Dec 3**

**Comment by smilefreak** [comment]
Postponed until next year, Peter Maxwell from NeSI will be giving a presentation.

**Comment by methylnick** [comment]
I figure you all might appreciate this. From a popular comic series, PhD comics. 

https://twitter.com/SantaCruzBio/status/671757157386207232/photo/1
